test/ClangImporter/requires_super_attribute.swift

```diff
@@ -0,0 +1,21 @@
+// RUN: %target-swift-frontend -typecheck -import-objc-header %S/Inputs/objc-requires-super-obj.h %s -verify-ignore-unknown
+// REQUIRES: OS=macosx
```

Comment:
I believe all this requires is objc_interop.

Reply:
Fixed!